ocs_ci/framework/conf/default_config.yaml

```diff
@@ -202,6 +202,9 @@ ENV_DATA:
   # ACM HUB deployment
   deploy_acm_hub_cluster: false
 
+  #Managed service - Managed StorageCluster size
+  size: '1'
```

Comment:
Please add unit to the description (not sure, if it is `TiB`).

I think that it would be better to use more specific name here (to make it clear, that it is related only to Managed Service) - but it was introduced in some previous PR, so it doesn't make sense to change it here.